doc/whats_new.rst

```diff
@@ -19,6 +19,8 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Add support for indexing and slicing :class:`mne.Annotations` by `Joan Massich`_
```

Comment:
Also mention iterating, i.e. "Add support for indexing, slicing, and iterating :class:`mne.Annotations` by `Joan Massich`_"

Reply:
+1